tests/Unit/EventTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Tests\Unit;
+
+use MelhorEnvio\MelhorEnvioSdkPhp\Event;
+use Tests\TestCase;
+
+class EventTest extends TestCase
+{
+    /**
+     * @test
+     * @small
+     */
+    public function it_can_register_and_trigger_the_callback(): void
```

Comment:
Acha que fica mais claro ?

```suggestion
    public function it_triggers_a_registered_callback(): void
```

Reply:
Feito!